projects/docs/src/app/docs-wrapper/pages/tooltip-page/tooltip-page.component.html

```diff
@@ -47,5 +59,18 @@
                 <option *ngFor="let item of themeList" [value]="item.value">{{ item.label }}</option>
             </select>
         </label>
+        <label class="w-100">
+            Cascade
+            <p><small>Add mode flexible placements to the tooltip, default is false</small></p>
```

Comment:
I can't seem to understand this sentence

Reply:
added more description about the positions